### PR TITLE
feat(pci): add black friday offer

### PR DESCRIPTION
**Pull request by frenautvh**
See : MANAGER-3932

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/new/payment/payment.scss [hunk 1]
```diff
@@ -35,5 +35,15 @@ pci-project-new-payment { /* stylelint-disable-line */
...
+
+    .black-friday {
+      color: black;
+      font-size: 14px;
+      .oui-status {
+        background-color: black;
+        color: white;
+        line-height: 1rem;
+      }
+    }
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/new/payment/payment.scss]: I suppose there were no other choice than custom styles ?

File: packages/manager/modules/pci/src/projects/new/payment/payment.html [hunk 2]
```diff
@@ -67,6 +67,13 @@
                         <span data-translate="pci_project_new_payment_use_credits"
                               data-translate-values="{ href: $ctrl.getStateLink('credits') }">
                         </span>
+                        <p class="black-friday mt-2">
```
Comment (marie-j) [inline packages/manager/modules/pci/src/projects/new/payment/payment.html]: Maybe you could have created a reusable component as you duplicate 
Also how come you have a custom style for the new project but not for credits ?

File: packages/manager/modules/pci/src/projects/new/payment/payment.scss (hunk 1 above)
Comment (antleblanc) [reply packages/manager/modules/pci/src/projects/new/payment/payment.scss]: Nope!
see: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/status

File: packages/manager/modules/pci/src/projects/new/payment/payment.html (hunk 2 above)
Comment (frenautvh) [reply packages/manager/modules/pci/src/projects/new/payment/payment.html]: credit is a child element of project that's why the style is only specified once

File: packages/manager/modules/pci/src/projects/new/payment/credits/credits.html [hunk 3]
```diff
@@ -41,6 +41,13 @@
              data-translate="pci_projects_new_credits_amount_min"
              data-translate-values="{ amount: $ctrl.creditMinPrice.text }">
         </div>
+        <p class="black-friday mt-2">
+            <span class="oui-status mr-1"
+                  data-translate="pci_project_new_payment_black_friday">
+            </span>
+            <strong data-translate="pci_project_new_payment_black_friday_info">
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/new/payment/credits/credits.html]: Looks like this is displayed even if `$ctrl.isBlackFridayAsteriskVisible()` is `false` ?

Also, this is not a11y friendly, you could either:
- add a link to the end note (and a link to go back here 😄 )
- add the note as a `aria-label`
- have a link to the note as a `aria-describedby`

I think the best solution is the link, what do you think ? 😄

Comment (frenautvh) [reply packages/manager/modules/pci/src/projects/new/payment/credits/credits.html]: i used aria-describedby

File: packages/manager/modules/pci/src/projects/project/vouchers/vouchers.html [hunk 4]
```diff
@@ -44,4 +45,5 @@
         </oui-column>
         <oui-column data-title="'cpb_vouchers_available_credit_cell' | translate" data-property="available_credit.text" data-sortable></oui-column>
     </oui-datagrid>
+    <pci-project-black-friday-notice class="mt-4 d-block"></pci-project-black-friday-notice>
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/vouchers/vouchers.html]: Is the `d-block` helper really needed?

We tend to apply margin in the "same" direction.

So I would suggest you to add this margin directly on the `<oui-datagrid>` (`mb-4`)

**Pull request merged**